examples/catsValidation/src/io/github/iltotore/iron/catsValidation/Main.scala

```diff
@@ -3,12 +3,10 @@ package io.github.iltotore.iron.catsValidation
 import cats.data.EitherNel
 import cats.effect.{IO, IOApp}
 import cats.syntax.all.*
-import io.github.iltotore.iron.{given, *}
+import io.github.iltotore.iron.*
 import io.github.iltotore.iron.cats.*
-import io.github.iltotore.iron.constraint.numeric.given
+import io.github.iltotore.iron.constraint.collection.Contain
```

Comment:
That's usually my preference as well 👍🏽 How do you want to address this? Documentation and examples showing this way?

Reply:
> Documentation and examples showing this way?

I think this is the right thing to do but I think it should not be done in this PR because it's unrelated.

I already have to polish some parts of the documentation so I will do it later.